packages/contracts-bedrock/test/L2/SequencerFeeVault.t.sol

```diff
@@ -25,14 +28,13 @@ contract SequencerFeeVault_Test is CommonTest {
         recipient = deploy.cfg().sequencerFeeVaultRecipient();
     }
 
-    /// @dev Tests that the minimum withdrawal amount is correct.
-    function test_minWithdrawalAmount_succeeds() external view {
-        assertEq(sequencerFeeVault.MIN_WITHDRAWAL_AMOUNT(), deploy.cfg().sequencerFeeVaultMinimumWithdrawalAmount());
-    }
-
     /// @dev Tests that the l1 fee wallet is correct.
     function test_constructor_succeeds() external view {
         assertEq(sequencerFeeVault.l1FeeWallet(), recipient);
+        assertEq(sequencerFeeVault.recipient(), recipient);
+        assertEq(sequencerFeeVault.MIN_WITHDRAWAL_AMOUNT(), deploy.cfg().sequencerFeeVaultMinimumWithdrawalAmount());
+        assertEq(sequencerFeeVault.minWithdrawalAmount(), deploy.cfg().sequencerFeeVaultMinimumWithdrawalAmount());
+        assertEq(uint8(sequencerFeeVault.withdrawalNetwork()), uint8(FeeVault.WithdrawalNetwork.L1));
```

Comment:
We test equality betwen the `MIN_WITHDRAWAL_AMOUNT` and `minWithdrawalAmount` getters, and should do the same for recipient and withdrawalNetwork

Reply:
I do now see these equality tests in `FeeVault_Test`, so will let you resolve this one however you feel appropriate

Reply:
decided to add tests for each getter in `SequencerFeeVault_Test` as in `FeeVault_Test`